packages/server/postgres/queries/decrementFreeTemplatesRemaining.ts

```diff
@@ -0,0 +1,13 @@
+import getPg from '../getPg'
+import {decrementFreeRetroTemplatesRemainingQuery} from './generated/decrementFreeCustomRetroTemplatesRemainingQuery'
+import {decrementFreePokerTemplatesRemainingQuery} from './generated/decrementFreeCustomPokerTemplatesRemainingQuery'
+
+const decrementFreeTemplatesRemaining = async (userId: string, templateType: 'retro' | 'poker') => {
```

Comment:
-1 pg-typed is deprecated. can we use kysely instead here?

Reply:
@nickoferrall thoughts on changing this to kysely?

Reply:
Ah I've updated the migration to kysely, but not the query. I can do this too

Reply:
Updated!